protoc-gen-openapiv2/internal/genopenapi/naming.go

```diff
@@ -0,0 +1,111 @@
+package genopenapi
+
+import (
+	"reflect"
+	"strings"
+)
+
+// LookupNamingStrategy looks up the given naming strategy and returns the naming
+// strategy function for it. The naming strategy function takes in the list of all
+// fully-qualified proto message names, and returns a mapping from fully-qualified
+// name to OpenAPI name.
+func LookupNamingStrategy(strategyName string) func([]string) map[string]string {
+	switch strategyName {
```

Comment:
Maybe use `strings.ToLower` here so users can also use capitals (for whatever reason).
```suggestion
	switch strings.ToLower(strategyName) {
```